### DISCUSSION TITLE
Transparent picture

### DISCUSSION THREAD

**Discussion by alexlimon404** [nutgram/nutgram#547] [Q&A]
Hello. I want to send a picture with transparency and text so that it would look like this
![image](https://github.com/nutgram/nutgram/assets/32906059/d6b7747e-9048-49c4-b79b-7a9d1aee7ec0)

My code
`$bot->sendPhoto(
                photo: 'https://ucarecdn.com/95dd55e1-0b78-46a3-88f2-fcaa1300bbe5/',
                caption: 'Text text text',
                clientOpt: [
                    'photo' => [
                        'width' => 10,
                        'height' => 10,
                    ]
                ]
            );
`
I received
![image](https://github.com/nutgram/nutgram/assets/32906059/5b90534f-91cf-474b-87d9-0c23f8c1cae9)

**Comment by Lukasss93** [comment]
Telegram doesn't support transparent images

**Reply by alexlimon404** [reply]
How to resize? width and height ?
